tasks/bin/run_external_test

```diff
@@ -13,7 +13,11 @@ git_thing = ARGV.shift
 
 build_dir = File.expand_path(ENV["TRAVIS_BUILD_DIR"] || Dir.pwd)
 
-env = { "GEMFILE_MOD" => "gem 'chef', path: '#{build_dir}'; gem 'ohai', git: 'https://github.com/chef/ohai.git', branch: 'master'" }
+env = {
+  "GEMFILE_MOD" => "gem 'chef', path: '#{build_dir}'; " \
+    "gem 'ohai', git: 'https://github.com/chef/ohai.git'",
+  "CHEF_LICENSE" => "accept-no-persist",
```

Comment:
should this just go into a global variable in the travis configs?   this whole bit of code is hacky and probably shouldn't be extended.

Reply:
It is set as a global variable in the travis config. But these tests must run in some kind of sandbox environment because I needed to set it here too